src/appleseedmaya/renderglobalsnode.cpp

```diff
@@ -333,31 +342,37 @@ MStatus RenderGlobalsNode::initialize()
     // Global bounces.
     m_globalBounces = numAttrFn.create("bounces", "bounces", MFnNumericData::kInt, 8, &status);
     numAttrFn.setMin(0);
+    numAttrFn.setSoftMax(32);
```

Comment:
Given that the hard bound is 100, does it make sense to have a soft bound at 32?

Reply:
Well, the slider bar is automagically rescaled to fit the new range if the user enters a value outside the soft min/max in the numeric field. What value would you suggest?

Reply:
I was thinking that since the full range is rather limited (being [0,100]), it didn't make sense to provide soft limits that were more restrictive than hard limits, but maybe I'm wrong.

Reply:
So removing the soft limits altogether and just leaving the range at [0,100]? It's a bit tricky to slide into the desired value. With the smaller range it's easier and the user can always enter a value in the numeric field, causing the slider bar to automatically rescale to the new range. I can add an annotation (mouse-hover text) and/or status bar message reminding the user the range is [0,100].

Reply:
You're right, it's probably a bad idea to change the range to [0, 100]. I think it's fine as it is actually.